client/a8c-for-agencies/components/sidebar/header/profile-dropdown.tsx

```diff
@@ -172,6 +177,8 @@ const ProfileDropdown = ( { compact, dropdownPosition = 'down' }: ProfileDropdow
 					</div>
 				) }
 			</Button>
+
+			{ withHelpCenter && <SidebarHelpCenter onClick={ () => {} } /> }
```

Comment:
Can we remove or make the OnClick optional on the `SidebarHelpCenter` instead of doing it?